### ISSUE TITLE
Cannot read an exported Keras model in GoCV

### ISSUE THREAD

**Issue by glarchev** [hybridgroup/gocv#617]
I'm trying to read a standard exported Keras model in GoCV. Here's my Python code that generates the protobuf file from the Keras model:

`model = tensorflow.keras.applications.MobileNet()
 tensorflow.saved_model.save(model, "myfolder")`

This generates `myfolder/saved_model.pb` file.

Here's my Go code for reading the model:

`
package main

import (
	"fmt"
	"gocv.io/x/gocv"
)

func main() {
	modelname := "myfolder/saved_model.pb"
	net := gocv.ReadNetFromTensorflow(modelname)
	if net.Empty() {
		fmt.Println("Error reading model")
	} else {
	         fmt.Println("Read model successfully!")
        }
	net.Close()
}
`

And here's the error that I'm getting:
`
[libprotobuf ERROR /tmp/opencv-20200105-75862-e9k4tb/opencv-4.2.0/3rdparty/protobuf/src/google/protobuf/wire_format_lite.cc:629] String field 'opencv_tensorflow.FunctionDef.Node.ret' contains invalid UTF-8 data when parsing a protocol buffer. Use the 'bytes' type if you intend to send raw bytes. 
libc++abi.dylib: terminating with uncaught exception of type cv::Exception: OpenCV(4.2.0) /tmp/opencv-20200105-75862-e9k4tb/opencv-4.2.0/modules/dnn/src/tensorflow/tf_io.cpp:42: error: (-2:Unspecified error) FAILED: ReadProtoFromBinaryFile(param_file, param). Failed to parse GraphDef file: myfolder/saved_model.pb in function 'ReadTFNetParamsFromBinaryFileOrDie'
`

- OS: MacOS
- OpenCV Version: 4.2.0
- How did you install OpenCV: `brew`
- Go Version: `go version go1.11.2 darwin/amd64`

**Comment by Danile71** [comment]
Try
``` ReadNet(file,"")```

**Comment by glarchev** [comment]
Tried it, same exact error.

**Comment by Danile71** [comment]
@glarchev, [https://answers.opencv.org/question/198805/how-to-load-a-keras-model-build-with-tensorflow-backend-in-opencv/](your) problem in opencv forum

**Comment by glarchev** [comment]
Yeah, I eventually exported the model as a frozen graph protobuf, but that introduced its own set of problems (which I'll describe in a separate bug report). I agree with the comment in that forum that Keras->Tensorflow->OpenCV conversion is shaky, which is why I was really hoping that the Keras->OpenCV solution would work.

**Comment by deadprogram** [comment]
I tried some similar approaches, and had no luck.

Likewise, make some apparent progress but not quite enough. I got my frozen graph to load, but I was not able to get any coherent results out of it. Of course, that might have been further user error on my part.
